modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -84,6 +84,34 @@ test.describe('Manage object definitions through Model Builder', () => {
 
 		// Clean up
 
+		await apiHelpers.objectAdmin.deleteObjectDefinition(
+			objectDefinition.id
+		);
+	});
+	test('show object definition details in "RightSidebar" after create object definition', async ({
+		apiHelpers,
+		modalAddObjectDefinitionPage,
+		modelBuilderPage,
+	}) => {
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		const objectDefinitionLabel = 'ObjectDefinitionLabel' + getRandomInt();
+
+		modelBuilderPage.createNewObjectDefinitionButton.click();
+
+		const objectDefinition =
+			await modalAddObjectDefinitionPage.createObjectDefinition(
+				objectDefinitionLabel
+		);
+
+		await expect(
+				modelBuilderPage.rightSidebar.filter({
+					hasText: objectDefinitionLabel + ' Details',
+				})
+		).toBeVisible();
```

Comment:
Hi @rafaellajordao, could you try using the getByTitle() function instead of filtering the right sidebar title

Reply:
I'll fix it!